pennylane/tape/tape.py

```diff
@@ -1103,7 +1103,9 @@ def to_openqasm(self, wires=None, rotations=True):
                     op.inv()
 
         # decompose the queue
-        operations = tape.expand(stop_at=lambda obj: obj.name in OPENQASM_GATES).operations
+        operations = tape.expand(
+            depth=10, stop_at=lambda obj: obj.name in OPENQASM_GATES
```

Comment:
We have one more level of "nesting", so need to define depth here. I never know how deep one should make this, but 10 should be more than enough for now